doc/dev/release-process.rst

```diff
@@ -2,172 +2,105 @@
   Ceph Release Process
 ======================
 
-1. Build environment
-====================
+Prequisites
+===========
 
-There are multiple build environments, debian based packages are built via pbuilder for multiple distributions.  The build hosts are listed in the ``deb_hosts`` file, and the list of distributions are in ``deb_dist``.  All distributions are build on each of the build hosts.  Currently there is 1 64 bit and 1 32 bit build host.
+Signing Machine
+---------------
+The signing machine is a virtual machine in the `Sepia lab<https://wiki.sepia.ceph.com/doku.php?id=start>`_.  SSH access is limited to the usual Infrastructure Admins along with a few other component leads (e.g., nfs-ganesha, ceph-iscsi).
```

Comment:
Suggest s/virtual machine/VM/ to be less stilted.

Reply:
If someone doesn't know what a triode is I don't want them using my software in the first place, but ok.  Maybe instead replace "signing machine" with "signing host"?